_static/example_files/purchase.order_functional_error_line_cant_adpat.csv

```diff
@@ -0,0 +1,6 @@
+"Order Reference","Supplier","Destination","Pricelist","Order Lines / Product","Order Lines / Quantity"

```

Comment:
Maybe you want to rename your file into purchase.order_functional_error_line_cant_**adapt**.csv ;-)